addon/components/plaid-line/component.js

```diff
@@ -44,8 +47,8 @@ const PlaidLineComponent = Ember.Component.extend(GroupElement, {
   curve: curveMonotoneX,
 
   didRender() {
-    const pathAttrs = this.getProperties(w('stroke strokeWidth strokeOpacity fill fillOpacity'));
-    const line = this.selection.select('path.line');
```

Comment:
All `const` does is prevent reassignment, it does not prevent mutation (contrary to most developers' understanding, and also contrary to the meaning of `const` in several other popular programming languages). 

In this case, I'm doing this for a few reasons
* It's the convention within ember and [ember-cli](https://github.com/ember-cli/ember-cli/issues/5091) codebases
* [It's in the default JSCS ruleset that ember-suave ships with](https://github.com/DockYard/ember-suave/pull/77)
* I agree w/ the reasoning behind the decisions driving points (1) and (2) :)